packages/esm-patient-vitals-app/src/vitals/vitals-overview.scss

```diff
@@ -106,3 +106,12 @@
   justify-content: end;
 }
 
+.customTable  tbody  tr:nth-child(even)  td {
+  background-color: #f4f4f4;
```

Comment:
Same here - why repeat? Can we make this reusable?

Reply:
Yes